src/components/header/user-menu/UserMenu.tsx

```diff
@@ -1,19 +1,22 @@
 import { DownOutlined, EyeOutlined, LogoutOutlined, SettingOutlined, UserSwitchOutlined } from '@ant-design/icons';
 import { User } from '@enigma-laboratory/shared';
 import { Avatar, Button, Divider, Dropdown, Flex, Popover, Space, Typography } from 'antd';
-import { IconMoon, IconSun } from 'assets/icons';
-import { Languages, Mode, useConfigProvider } from 'contexts';
 import { ReactElement } from 'react';
 import { useTranslation } from 'react-i18next';
-import { generateColorFromAlphabet } from 'utils';
+
+import { IconMoon, IconSun } from 'assets/icons';
+import { Languages, Mode, useConfigProvider } from 'contexts';
+import i18n from 'contexts/config-provider/i18n';
```

Comment:
@phanvohieunghia  , I think i18n should be moved to translation service
export {i18n} from 'service'

Reply:
because i18n is exported "defualt" key